app/javascript/packs/stylesheets/custom_table.scss

```diff
@@ -54,3 +54,21 @@
 .table-smaller-font {
   font-size: 0.9em;
 }
+
+.patients-table .table-hover tbody tr:hover {
```

Comment:
can we nest these?

Reply:
also do you need `.table-hover` in here?

Reply:
Nope, turns out you dont! I've nested them too.